conformance/setup.go

```diff
@@ -510,8 +512,10 @@ func init() {
 	automaticCrossmountVal, runAutomaticCrossmountTest = os.LookupEnv(envVarAutomaticCrossmount)
 	automaticCrossmountEnabled, _ = strconv.ParseBool(automaticCrossmountVal)
 
-	reportJUnitFilename = "junit.xml"
-	reportHTMLFilename = "report.html"
+	if dir := os.Getenv(envVarReportDir); dir != "none" {
+		reportJUnitFilename = filepath.Join(dir, "junit.xml")
+		reportHTMLFilename = filepath.Join(dir, "report.html")
+	}
```

Comment:
The "none" is throwing an error for me because the code expects the value to always be defined. I'd recommend changing the logic slightly:

```suggestion
	reportJUnitFilename = "junit.xml"
	reportHTMLFilename = "report.html"
	if dir := os.Getenv(envVarReportDir); dir != "" {
		reportJUnitFilename = filepath.Join(dir, "junit.xml")
		reportHTMLFilename = filepath.Join(dir, "report.html")
	}
```

Reply:
> The "none" is throwing an error for me because the code expects the value to always be defined.

Ah, I hadn't noticed that warning message (it's not actually an error because so it didn't cause the tests to fail for me), sorry!

I've pushed an alternative fix which still keeps the "none" capability rather than forcing a write to a temporary file: instead I changed `HTMLReporter.endSuite` so that it doesn't try to write the file if it's empty, mimicking the behavior of `github.com/onsi/ginkgo/v2/types.ReporterConfig.JUnitReport`. That way the warning is no longer displayed.

Reply:
Looks good, and that clears up the error I was seeing. Lets document the new variable under https://github.com/opencontainers/distribution-spec/tree/main/conformance#html-report

Reply:
Great. I updated that doc but, after a moment of consideration, chose a slightly different place to make the change as `OCI_REPORT_DIR` pertains to both the HTML report file and the JUnit report file, so I thought it was probably more appropriate to document where those files are both mentioned.

PTAL.